eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -101,6 +101,9 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('security.yml');
         // Slots
         $loader->load('slot.yml');
+        if (interface_exists('FOS\JsRoutingBundle\Extractor\ExposedRoutesExtractorInterface')) {
```

Comment:
I'm replacing it with `if ($container->hasExtension('fos_js_routing'))`. No objections ?